src/shipit_code_coverage/shipit_code_coverage/uploader.py

```diff
@@ -25,12 +26,12 @@ def coveralls(data):
     return result['url'] + '.json'
 
 
-def codecov(data, commit_sha, token, flags=None):
+def codecov(data, commit_sha, flags=None):
     logger.info('Upload report to Codecov')
 
     params = {
         'commit': commit_sha,
-        'token': token,
+        'token': secrets[secrets.CODECOV_TOKEN],
```

Comment:
I would have implemented `__getattr__` in order to simply do `secrets.CODECOV_TOKEN`.